site/src/atomics/display.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+Display atomics are used to set the display behavior of an element.
+
+## Class Pattern
+
+`display-<value>-<screensize>`
+
+For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md).
+
+Accepted values
+
+- `block`, `flex`, `inline`, `inline-block`, `inline-flex`, `grid`, `none`
+
+Accepted screensizes
+
+- `tablet, desktop, widescreen`
+
+## Usage
+
+```html
+<div class="display-block border">
+	<div>Foo</div>
+	<div>Foo</div>
+</div>
+<div class="display-flex-tablet border">
+	<div>Bar</div>
+	<div>Bar</div>
+</div>
+<div class="display-inline-desktop border">
+	<div>Baz</div>
+	<div>Baz</div>
+</div>
+<div class="display-inline-flex-widescreen border">
+	<div>Foo</div>
+	<div>Foo</div>
+</div>
+```
```

Comment:
The above looks great, should we wait until the code block changes hit and add an `Available classes` section at the end of this?

Reply:
Yes, I'll need to revisit after the code block changes are checked in.

Reply:
What do you think about splitting one large usage example into 4 smaller ones?

In this particular case, it feels hard for me to read the examples: different position classes are assigned to blocks, as well as different screen sizes, and it all make examples to be too close to each other and hard to understand what class is assigned to what section 😵

![image](https://user-images.githubusercontent.com/57374379/119401787-daf90f00-bc90-11eb-8371-939225b4c4f1.png)

Reply:
I think that makes sense, updated.